sdk/python/testvectors/utils.py

```diff
@@ -0,0 +1,11 @@
+import sha3
+
+from symbolchain.CryptoTypes import Hash256
+from symbolchain.symbol.MerkleHashBuilder import MerkleHashBuilder
+
+
+def calculate_transactions_hash(transactions):
```

Comment:
is this used anywhere?

Reply:
explained, transaction hashes in txes are currently invalid 😬